test/integration/corechecks/docker/exitcode_test.go

```diff
@@ -0,0 +1,44 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+package docker
+
+import (
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/metrics"
+)
+
+func init() {
+	registerComposeFile("exitcode.compose")
+}
+
+func TestContainerExit(t *testing.T) {
+	expectedTags := []string{
+		instanceTag,
+		"docker_image:busybox:latest",
+		"image_name:busybox",
+		"image_tag:latest",
+		"highcardlabeltag:exithigh",
+		"lowcardlabeltag:exitlow",
+		"highcardenvtag:exithighenv",
+		"lowcardenvtag:exitlowenv",
+	}
+	sender.AssertServiceCheck(t, "docker.exit", metrics.ServiceCheckOK, "", expectedTags, "Container exitcode_exit0_1 exited with 0")
+	for _, check := range []metrics.ServiceCheckStatus{metrics.ServiceCheckWarning, metrics.ServiceCheckCritical, metrics.ServiceCheckUnknown} {
+		sender.AssertServiceCheckNotCalled(t, "docker.exit", check, "", expectedTags, "Container exitcode_exit0_1 exited with 0")
```

Comment:
Good to test that. To cover a broader scope of calls AssertNotCalled, I'd only assert:
  - "docker.exit"
  - check  (to rename `status` for readability btw)
  - AnythingOfType("string")
  - just []string{"container_name:xxxxx"}
  - AnythingOfType("string")


what do you think?